src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -21,6 +21,7 @@
  */
 public class ModelManager implements Model {
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
+    private static final Integer HOURS_PER_SHIFT = 6;
```

Comment:
Minor issue but 8 hours may be more realistic since McDonald's usually open for more than 12 hours a day